src/components/asset-sidebar/index.tsx

```diff
@@ -45,16 +53,19 @@ export const AssetSideBar = ({ assetDetails, alerts, ...properties }: Props) =>
               {assetAddress.uprn ? "Edit address details" : "Cannot edit: UPRN missing"}
             </Button>
           )}
-          <CautionaryAlertsDetails alerts={alerts} />
-          <TenureDetails tenure={tenure} />
+          {showCautionaryAlerts && <CautionaryAlertsDetails alerts={alerts} />}
```

Comment:
Here you have a refactor which isn't necessarily part of the description - you're not showing the CC header if there are no Cautionary Contacts. is that correct?